project/GenScalaTestNative.scala

```diff
@@ -202,14 +194,14 @@ object GenScalaTestNative {
       "org/scalatest/time" -> List.empty, 
       "org/scalatest/words" -> List.empty, 
       "org/scalatest/enablers" -> List.empty, 
-      "org/scalatest/funsuite" -> asyncs("jvm/funsuite/src/main/scala/org/scalatest/funsuite"), 
-      "org/scalatest/featurespec" -> asyncs("jvm/featurespec/src/main/scala/org/scalatest/featurespec"), 
-      "org/scalatest/funspec" -> asyncs("jvm/funspec/src/main/scala/org/scalatest/funspec"), 
-      "org/scalatest/freespec" -> asyncs("jvm/freespec/src/main/scala/org/scalatest/freespec"), 
-      "org/scalatest/flatspec" -> asyncs("jvm/flatspec/src/main/scala/org/scalatest/flatspec"), 
+      "org/scalatest/funsuite" -> List.empty,
+      "org/scalatest/featurespec" -> List.empty,
+      "org/scalatest/funspec" -> List.empty,
+      "org/scalatest/freespec" -> List.empty,
+      "org/scalatest/flatspec" -> List.empty,
       "org/scalatest/prop" -> List.empty, 
-      "org/scalatest/propspec" -> asyncs("jvm/propspec/src/main/scala/org/scalatest/propspec"), 
-      "org/scalatest/wordspec" -> asyncs("jvm/wordspec/src/main/scala/org/scalatest/wordspec"), 
+      "org/scalatest/propspec" -> List.empty,
+      "org/scalatest/wordspec" -> List.empty,
       "org/scalatest/concurrent" -> (List(
         "Waiters.scala",        // skipeed because doesn't really make sense on js's single-thread environment.
```

Comment:
s/skipeed/skipped/

Scalatest devos: Yes, please, this PR would make using Scalatest with Scala Native easier.
It appears to me to be Scala Native specific, so it should not perturb other platforms.

Thank you for considering it, especially with your busy workload.

Lee, an individual SN contributor, pecking away at hoped-for/eventual async support.

Reply:
@LeeTibbert I just uncommented this lines, the same typo exists for example here: https://github.com/scalatest/scalatest/blob/f3d5f9d66631dc49b2218c8edd039840788bd915/project/GenScalaTestJS.scala#L225